test/parallel/test-domain-run.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const domain = require('domain');
+
+const d = new domain.Domain();
+
+assert.strictEqual(d.run(() => 'return value'),
```

Comment:
Here too, keeping or improving on the comments of the previous version of this test would help understand its purpose.